ui/pages/settings/experimental-tab/experimental-tab.component.tsx

```diff
@@ -272,6 +279,17 @@ export default class ExperimentalTab extends PureComponent<ExperimentalTabProps>
           toggleOffLabel: t('off'),
           toggleOnLabel: t('on'),
         })}
+        {this.renderToggleSection({
+          title: t('bitcoinTestnetSupportToggleTitle'),
+          description: t('bitcoinTestnetSupportToggleDescription'),
+          toggleValue: bitcoinTestnetSupportEnabled,
+          toggleCallback: (value) => {
```

Comment:
Just to double check: do we want to track the creation of testnet accounts?

Reply:
I haven't asked to anyone to be honest, but since it was mostly dev/test oriented, I thought I would just not add any events.

If we were to do that, we would need to adjust our schemas or probably have a new event just for this...

Let's discuss this internally!

Reply:
As discussed together with the team, I'll add metrics support even for testnet accounts! Thanks